shvutil.c

```diff
@@ -138,7 +138,7 @@ ShUtilvAllocateContiguousMemory(
 	// If we're here then we'll just fall back to using executable
 	// memory.
 	//
-#pragma warning(suppress: 30029)
+#pragma warning(suppress: 30030)
```

Comment:
You'll want to use
https://msdn.microsoft.com/en-us/library/gg155764(v=winembedded.70).aspx
instead of warning, use prefast supress.

Then use the header \shared\suppress.h to see if the warning is there (it
might not be).

Best regards,
Alex Ionescu

On Mon, Apr 18, 2016 at 11:06 AM, Joe Sylve <notifications@github.com>
wrote:

> In shvutil.c
> <https://github.com/ionescu007/SimpleVisor/pull/8#discussion_r60105698>:
>
> > @@ -138,7 +138,7 @@ ShUtilvAllocateContiguousMemory(
> >  	// If we're here then we'll just fall back to using executable
> >  	// memory.
> >  	//
> > -#pragma warning(suppress: 30029)
> > +#pragma warning(suppress: 30030)
>
> Sorry, didn't mean to PR these changes (still getting used to the github
> interface), but thanks for the comment.
>
> I can't find that header, and can you give an example of using the comment
> feature? I don't see that listed at
> https://msdn.microsoft.com/en-us/library/2c8f766e.aspx
>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
> <https://github.com/ionescu007/SimpleVisor/pull/8/files/12cd63315d4eda0f05634552cad1b3760c911bb9..66972d70af186e0b7883d82f1740123360b75571#r60105698>
>